RELEASING.md

```diff
@@ -12,6 +12,17 @@
     * Merge the one targeted towards the release branch.
     * The builds will fail for the `main` pr because of validation rules. Follow the [release workflow](https://github.com/open-telemetry/opentelemetry-python/blob/main/RELEASING.md) for the core repo up until this same point. Change the SHAs of each PR to point at each other to get the `main` builds to pass.
 
+### Preparing a major or minor release for individual package
+
+> [!NOTE]
+> Per-package release is supported only for packages included in the corresponding workflow. Libraries that support per-package release are currently
```

Comment:
Maybe we can add a section in RELEASING.md outlining which packages are supported and link to there from this note?